Cesium3DTiles/include/Cesium3DTiles/Tileset.h

```diff
@@ -417,6 +417,17 @@ class CESIUM3DTILES_API Tileset final {
     return this->_supportsRasterOverlays;
   }
 
+  /**
+   * @brief Returns the value indicating the glTF up-axis.
+   *
+   * This function is not supposed to be called by clients.
+   *
+   * The value indicates the axis, via 0=X, 1=Y, 2=Z.
+   *
+   * @return The value representing the axis
+   */
+  uint8_t getGltfUpAxis() const noexcept { return this->_gltfUpAxis; }
```

Comment:
maybe returning an enum is a better option since it makes the API more clear

Reply:
Yes, I considered adding a full-fledged enum/class like https://github.com/CesiumGS/cesium/blob/master/Source/Scene/Axis.js , but wasn't sure where to put it (because it would be "public"). But ( also @kring ) it would probably make sense to add an `Axis` enum in `CesiumGeometry`, right?

Reply:
That sounds fine to me.